packages/playwright/src/program.ts

```diff
@@ -415,6 +425,7 @@ const testOptions: [string, string][] = [
   ['--reporter <reporter>', `Reporter to use, comma-separated, can be ${builtInReporters.map(name => `"${name}"`).join(', ')} (default: "${defaultReporter}")`],
   ['--retries <retries>', `Maximum retry count for flaky tests, zero for no retries (default: no retries)`],
   ['--shard <shard>', `Shard tests and execute only the selected shard, specify in the form "current/all", 1-based, for example "3/5"`],
+  ['--show-warnings', `Show all warning messages collected in the last test run. Does not execute tests.`],
```

Comment:
- I wonder whether we want a mode where warnings are printed inline, and also fail the test run, for the strict mode on CI.
- How many warnings do we get for `npm run ctest`?

Reply:
I'm not seen any unawaited promise warnings anywhere in our repo. Just reran `ctest` to verify and no warnings were captured.